profile/README.md

```diff
@@ -1 +1,8 @@
-Welcome!
+## Welcome to the OpenVINO™ GitHub organization!
+[OpenVINO™](https://en.wikipedia.org/wiki/OpenVINO) is an open-source toolkit for optimizing and deploying AI inference.
```

Comment:
Maybe we could add some intro text, like "Here are some repositories you may find interesting:" ... or "useful" or "may want to check out"...